src/ext/java/org/opentripplanner/ext/fares/model/FareRuleSet.java

```diff
@@ -40,6 +40,15 @@ public Set<RouteOriginDestination> getRouteOriginDestinations() {
     return routeOriginDestinations;
   }
 
+  public boolean hasRules() {
```

Comment:
This one needs Javadoc and a test.

Reply:
Good point, I added tests for this and `matches`.

Reply:
That's a very nice test but where is the Javadoc?

Reply:
Oops